ote_sdk/ote_sdk/entities/dataset_item.py

```diff
@@ -93,7 +95,7 @@ def __init__(
         metadata: Optional[Sequence[MetadataItemEntity]] = None,
         subset: Subset = Subset.NONE,
         ignored_labels: Optional[
-            Union[List[LabelEntity], Tuple[LabelEntity, ...], Set[LabelEntity]]
```

Comment:
Why?

Reply:
If I use debugger, “Tuple[LabelEntity, ...].__dict__.get("__args__")” (to get list of expected types ) will return (<class 'ote_sdk.entities.label.LabelEntity'>, Ellipsis). This “Ellipsis” (second element of tuple) even not type-object. Does it acceptable as ignored_labels expected type?

Reply:
Tuple[LabelEntity, ...] is not the same as Tuple[LabelEntity] , see https://docs.python.org/3/library/typing.html#typing.Tuple

Reply:
Thanks, I will roll back changes in ignored_labels expected type, with actual version of input parameters validation recursion, this case is included and works as expected

Reply:
Thanks, now I understand.
@LeonidBeynenson for that case the check (argument_checks.py lines 126-129)
        'if len(nested_elements_class) != 1:
            raise TypeError(
                "length of nested expected types for Sequence should be equal to 1"
            )'
Does not work, should remove it, or add separate branch for tuple objects, ex raise TypeError is len(nested_elements_class) > 2?

Reply:
Rolled back changes in ignored_labels expected type for DatasetItemEntity, added branch for tuples length equal 2 in check_parameter_type function